src/avkit.cs

```diff
@@ -518,23 +631,54 @@ interface AVKitMetadataIdentifier {
 		NSString ServiceIdentifier { get; }
 	}
 
-	[TV (11,0), iOS (11,0)]
+	#endif
+	
+	[TV (11,0), iOS (11,0), Mac (10,15)]
+#if !MONOMAC
 	[BaseType (typeof (UIView))]
-	interface AVRoutePickerView {
-
+#else
+	[BaseType (typeof (NSView))]
+#endif
+	interface AVRoutePickerView
+	{
+#if !MONOMAC
 		[Export ("initWithFrame:")]
 		IntPtr Constructor (CGRect frame);
-
-		[NullAllowed, Export ("delegate", ArgumentSemantic.Weak)]
+#endif
+		[Wrap ("WeakDelegate")]
+		[NullAllowed]
```

Comment:
Actually there is a breaking change because of this modification : https://jenkins.mono-project.com/job/xamarin-macios-pr-builder/10962/API_20diff_20_28PR_20only_29/

What do you think about it?

Reply:
One solution here is to not change the `Delegate` property, and add `[Sealed]` to the new `WeakDelegate` property (it's only a problem to have two members with the same Objective-C selector they're both virtual):

```csharp
[NullAllowed, Export ("delegate", ArgumentSemenatic.Weak)]
[Sealed]
NSObject WeakDelegate { get; set; }
```